README.md

```diff
@@ -306,6 +306,28 @@ Generated shape:
 <http://www.ldbc.eu/ldbc_socialnet/1.0/vocabulary/Person> a rdfs:Class.
 ```
 
+#### Vocabuary Predicate Domain Handler
+
+Generates vocabulary information.
+
+```json
+{
+  "handlers": [
+     {
+      "@type": "EnhancementHandlerVocabularyDomain",
```

Comment:
```suggestion
      "@type": "EnhancementHandlerVocabularyPredicateDomain",
```

Reply:
It didn't get pushed earlier have now made the changes